### PR TITLE
Fix/doc 4319: Kill Shindig documentation

### PR DESCRIPTION
**Pull request by mzorai**

### CODE REVIEW

**Comment by mzorai** [comment]
I updated "what's new" section with changes done on 5.3. In develop the sections was the one done on 5.2

File: docs/Upgrade.rst [hunk 1]
```diff
@@ -58,6 +58,10 @@ out the :ref:`complete list <Upgrade.BreakingChanges.Templates>`. If
 your custom extension overrides some Groovy templates, you must check 
 if it has been changed, and update it if it is the case.
 
+**Shindig removal**
+
+WIth eXo Platform 5.3 version, we no more support gadgets as we dropped the use of `Shindig <https://shindig.apache.org/>`__
```
Comment (plamarque) [inline docs/Upgrade.rst]:
```suggestion
In eXo Platform 5.3 version, we no longer support as we dropped `Apache Shindig <https://shindig.apache.org/>` which has been retired.
```

File: docs/whatsnew.rst [hunk 2]
```diff
@@ -1,134 +1,143 @@
...
 
-**Platform RDBMS**
+We also upgraded UI layer with latest versions of Vue.js and Vuetify frameworks.
+We invite you to read the whole tutorial to develop portlets in eXo with Vuetify and Vue.js 
+in  :ref:`this tutorial <PLFDevGuide.DevelopingApplications.DevelopingPortlet.Vuetify>`.
 
-We continued to decrease our dependency on JCR 
-for data persistence. Now, the following components rely on the relational 
-**database** storage for better speed and manageability :
+With 5.3 version, eXo Platform opted to remove `Shindig <https://shindig.apache.org/>`__,
```
Comment (plamarque) [inline docs/whatsnew.rst]:
```suggestion
In version 5.3, eXo Platform decided to remove `Shindig <https://shindig.apache.org/>`__, 
```

**Review by plamarque — CHANGES_REQUESTED** [review]

File: docs/whatsnew.rst [hunk 3]
```diff
@@ -1,134 +1,143 @@
...
--  Product information
--  Login history
--  Wiki
--  Files
--  Social
--  Notifications
--  Email queue
--  Settings
+- The dependance to specific versions of some important libraries namely Guava and which is hard to upgrade
+- SSRF security issues exposure
```
Comment (thomasdelhomenie) [inline docs/whatsnew.rst]: I don't think we should expose security issues, since it is still available in older versions (5.2, ...).

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: docs/whatsnew.rst (hunk 3 above)
Comment (mzorai) [reply docs/whatsnew.rst]: Oh yes, you're right! thanks for the catch

Comment (mzorai) [reply docs/whatsnew.rst]: done

**Review by thomasdelhomenie — APPROVED** [review]

**Comment by mzorai** [comment]
@plamarque can you please approve to merge it ?

**Review by plamarque — CHANGES_REQUESTED** [review]
There are some images that don't seem related to shindig removal

**Comment by mzorai** [comment]
@plamarque In fact, at the beginning I had to update the "what's new" section with 5.3 updates as it was on 5.2 version. Branches update was done before we update "what's new" section for 5.3 that's why some images are included in this PR. I already reported this in a previous [comment](https://github.com/exoplatform/exo-documentation/pull/198#issuecomment-583386368)

**Review by plamarque — APPROVED** [review]

**Pull request merged**